src/lib.rs

```diff
@@ -290,6 +325,19 @@ struct Action {
     count: Option<AtomicUsize>,
 }
 
+impl Clone for Action {
+    fn clone(&self) -> Self {
+        Action {
+            count: self
+                .count
+                .as_ref()
+                .map(|c| AtomicUsize::new(c.load(Ordering::Relaxed))),
```

Comment:
This looks hacky to me, maybe `count` should be the original configured value.

Reply:
the local registry will inherit the global registry, for example, we just want a point named`GLOBAL_PANIC_ONCE` panic just once, so we must inherit the parameter from global registry